### PR TITLE
[WIP]Pad

### PR DESCRIPTION
**Pull request by absolutelyNoWarranty**

### CODE REVIEW

File: strpipe/toolkit/default_tokens.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-
 class DefaultTokens:
 
     sos = '<SOS>'
     eos = '<EOS>'
     unk = '<UNK>'
     pad = '<PAD>'
+    nul = '\0'  # For Cython classes to use as Python's 'None'.
```
Comment (SoluMilken) [inline strpipe/toolkit/default_tokens.py]: 喔喔喔喔喔喔喔

Comment (stegben) [reply strpipe/toolkit/default_tokens.py]: This is public repo, plz don't "喔" so many times...

File: strpipe/toolkit/pad_sentences.pxd [hunk 2]
```diff
@@ -1,13 +1,17 @@
 cdef list pad_sentence_in_c(
     list sentence,
     str pad_token,
+    str sos_token,
+    str eos_token,
     unsigned int maxlen,
```
Comment (SoluMilken) [inline strpipe/toolkit/pad_sentences.pxd]: @absolutelyNoWarranty 想問 你覺得 maxlen =-1 還是 maxlen = None 還是 maxlen = 0 才能代表 需要計算maxlen ??
如果 maxlen = -1 比較好 那我要改型別成int 不用 unsigned int 惹

File: strpipe/toolkit/pad_sentences.pyx [hunk 3]
```diff
@@ -1,10 +1,14 @@
+#cython: nonecheck=True
+from libc.stdio cimport printf
```
Comment (SoluMilken) [inline strpipe/toolkit/pad_sentences.pyx]: 為毛需要J個

**Comment by SoluMilken** [comment]
Please fix these error. 

```
flake8
./strpipe/toolkit/test/test_pad_sentences.py:85:5: E124 closing bracket does not match visual indentation
./strpipe/toolkit/test/test_pad_sentences.py:97:5: E124 closing bracket does not match visual indentation
./strpipe/toolkit/test/test_pad_sentences.py:109:5: E124 closing bracket does not match visual indentation
./strpipe/toolkit/test/test_pad_sentences.py:121:5: E124 closing bracket does not match visual indentation
./strpipe/toolkit/test/test_pad_sentences.py:133:5: E124 closing bracket does not match visual indentation
./strpipe/toolkit/test/test_pad_sentences.py:142:5: E128 continuation line under-indented for visual indent
5     E124 closing bracket does not match visual indentation
1     E128 continuation line under-indented for visual indent
make: *** [lint] Error 1
The command "make lint" exited with 2.
0.72s$ make test
pytest
============================= test session starts ==============================
platform linux -- Python 3.6.3, pytest-3.3.0, py-1.5.2, pluggy-0.6.0
rootdir: /home/travis/build/Yoctol/strpipe, inifile:
plugins: mock-1.10.0
collected 47 items / 1 errors                                                  
==================================== ERRORS ====================================
________________ ERROR collecting strpipe/ops/tests/test_pad.py ________________
strpipe/ops/tests/test_pad.py:80: in <module>
    Pad(sos_token='<START_TOK>'),
strpipe/ops/pad.pyx:28: in strpipe.ops.pad.Pad.__init__
    cdef int _maxlen
E   TypeError: __init__() got an unexpected keyword argument 'sos_token'
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.29 seconds ============================
```

**Comment by absolutelyNoWarranty** [comment]
@stegben Regression test 要怎麼處理？

**Comment by stegben** [comment]
wait, why closed?

**Pull request closed without merging**